modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/js/main.js

```diff
@@ -811,91 +811,107 @@ AUI.add(
 					var instance = this;
 
 					var cmdNode = instance.byId('cmd');
-					var portletURL;
 					var redirectNode = instance.byId('redirect');
 
 					if (cmdNode.val() === 'add' || cmdNode.val() === 'update') {
-						portletURL = Liferay.PortletURL.createURL(
-							redirectNode.val()
-						);
+						var params = {
+							cmd: cmdNode.val(),
+							exportConfigurationButtons: 'custom',
+							mvcRenderCommandName: 'editExportConfiguration',
+							publishConfigurationButtons: 'custom',
+							tabs2: 'new-export-process'
+						};
 
-						portletURL.setParameter('cmd', cmdNode.val());
+						var url = Liferay.Util.PortletURL.createPortletURL(
+							redirectNode.val(),
+							{
+								cmd: params.cmd
+							}
+						);
 
 						if (instance._exportLAR) {
-							portletURL.setParameter(
-								'mvcRenderCommandName',
-								'editExportConfiguration'
-							);
-							portletURL.setParameter(
-								'tabs2',
-								'new-export-process'
-							);
-							portletURL.setParameter(
-								'exportConfigurationButtons',
-								'custom'
+							url = Liferay.Util.PortletURL.createPortletURL(
```

Comment:
Instead of having many many `Liferay.Util.PortletURL.createPortletURL` calls, just have one, like in the original code. You can modify `params` object throughout code:
```
if (instance._exportLAR) {
    params.mvcRenderCommandName = 'editExportConfiguration';
    params.tabs2 = 'new-export-process';
    params.exportConfigurationButtons = 'custom';
} else {
```
...
```
url = Liferay.Util.PortletURL.createPortletURL(redirectNode.val(), params);
```